vectorTiling.py

```diff
@@ -0,0 +1,42 @@
+
+import json
+import click
+import subprocess
+import utils 
+import os 
+import logging
+ 
+@click.command()
+@click.argument('sources', type=click.Path(exists=True), required=True)
+@click.argument('output', type=click.Path(exists=True), required=True)
+@click.argument('min_zoom', default=5)
+@click.argument('max_zoom', default=14)
+def vectorTiling(sources, output, min_zoom, max_zoom):
+    """ Function that creates vector tiles
+    PARAMS: 
+    	- sources : directory where the geojson file(s) are
+    	- output : directory for the generated data 
+    """
+    files = []
+    for file in utils.get_files(sources):
+    	if not os.path.isdir(file): 
+    		if file.split('.')[1] == 'geojson': 
+    			files.append(file)
+
+	logging.info("{} geojson found".format(len(files)))
+	paths_string = ''
+
+    for file in files: 
```

Comment:
`file` is a builtin class in python. Using it as a variable name works, but it's not a good practice.